Example/Tests/Helpers/AnalyticsPluginConfig.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  AnalyticsPluginConfig.swift
+//  PlayKit
+//
+//  Created by Gal Orlanczyk on 08/02/2017.
+//  Copyright © 2017 CocoaPods. All rights reserved.
+//
+
+import Foundation
+
+enum AnalyticsPluginConfig {
```

Comment:
@noamtamim I am changing the name to `PluginTestConfiguration` it will hold all default configurations and static data ok?